README.md

```diff
@@ -124,7 +124,14 @@ Variables that may be desired to be overriden in the Terraform workspace:
 
   Valid values: any valid AWS KMS CMK ARN
   
-  Default: ``
+  Default: null
+
+* **sns_cmk_key_arn**
+  Optional ARN for the CMK that should be used for the AWS KMS encryption for Notifications SNS topic
```

Comment:
I would add a note that the customer needs to ensure that our Console & Agent roles are given access to this key in order to send messages to the SNS Topic

Reply:
Or it looks like you are attaching that policy in TF? If that's the case, we should call out that we will add the permissions for our roles to use the key